tests/TestFramework/DataFakerTest.php

```diff
@@ -824,4 +824,14 @@ public function testRiskScore()
         $this->assertGreaterThanOrEqual(-2, $score);
         $this->assertLessThanOrEqual(100, $score);
     }
+
+    /**
+     * @return void
+     */
+    public function testCancelReason()
+    {
+        $cancelReason = $this->faker->subscriptionCancelReason();
+        $this->assertTrue(is_string($cancelReason));
+        $this->assertGreaterThanOrEqual(0, (int) $cancelReason);
```

Comment:
Is the reason you're not checking that `$cancelReason < 1000` because we have/could add our own custom-defined codes?

Reply:
yeah. i figured it probably didn't matter. but i'm open to changing it.

Reply:
I think it's fine the way it is, I just wanted to make sure I understood